ModuleId = "C4-3E-6F-1E-2A-97-76-95-28-2A-87-C2-E9-E7-CD-1D-08-F2-14-3C"
hitPointId = 325
context = Null
exception = System.Collections.Generic.KeyNotFoundException: The given key '325' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at AltCover.Recorder.Counter.addSingleVisit(Dictionary`2 counts, String moduleId, Int32 hitPointId, Track context) in C:\projects\altcover\AltCover.Recorder\Base.fs:line 401
   at AltCover.Recorder.Instance.I.addVisit(String moduleId, Int32 hitPointId, Track context) in C:\projects\altcover\AltCover.Recorder\Recorder.fs:line 328
   at AltCover.Recorder.Instance.I.logException[a,b,c,d](a moduleId, b hitPointId, c context, d x)
   at AltCover.Recorder.Instance.I.addVisit(String moduleId, Int32 hitPointId, Track context)
   at AltCover.Recorder.Instance.I.visitImpl@395.Invoke(String moduleId, Int32 hitPointId, Track context)
   at AltCover.Recorder.Instance.I.visitImpl(String moduleId, Int32 hitPointId, Track context)
   at OOPListHW.List`1.get_Head()
   at OOPListHW.head[Value](IList`1 lst)
   at OOPListHW.bubble@61-1.Invoke(IList`1 lst)
   at OOPListHW.bubbleSort[Value](IList`1 lst)
   at MathMechLessons.Tests.OOPList.tests@141-43.Invoke(Unit _arg4)
   at Expecto.Impl.execTestAsync@569-1.Invoke(Unit unitVar)
   at Microsoft.FSharp.Control.AsyncPrimitives.CallThenInvoke[T,TResult](AsyncActivation`1 ctxt, TResult result1, FSharpFunc`2 part2)
   at Microsoft.FSharp.Control.Trampoline.Execute(FSharpFunc`2 firstAction)
   at <StartupCode$FSharp-Core>.$Async.clo@181-16.Invoke(Object o)
   at System.Threading.QueueUserWorkItemCallbackDefaultContext.Execute()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading.PortableThreadPool.WorkerThread.WorkerThreadStart()
   at System.Threading.Thread.StartCallback()

